appveyor.yml

```diff
@@ -1,44 +1,63 @@
-version: 1.73.{build}
+version: 1.86.{build}
```

Comment:
would it work with `version: {build}`?

Reply:
Not literally but https://github.com/danmar/cppcheck/pull/1566/commits/cb50c86cd271ba38acfea39a4d3e2bdccc1e522e seems to work out